serializers.py

```diff
@@ -758,6 +762,10 @@ def pre_process(self, data: dict) -> None:
 
         # Add to end of issue order, by default
         self.apply_default_value(data, "order", len(self.journal.issues))
+        show_year = data.pop('show_year', False)
+        year = data.pop('year', "0")
```

Comment:
I wonder about the default for year. Is zero or "0" or 2000?

Reply:
mostly I don't expect the default to be used because year would have to be missing but showYear set to true.  However, I think a more acceptable default would either be the year of date_published or current_year + 50 (used elsewhere and ultimate default)